### ISSUE TITLE
Azk is alive ? O Azk ainda está vivo ?

### ISSUE THREAD

**Issue by chiaretto** [azukiapp/azk#696]
O projeto tem quase 2 meses sem commits. 
O blog tem mais de 1 ano sem postagens.
O gitter tem 3 meses sem resposta oficial.

O projeto ainda está vivo ou foi abandonado ?

**Comment by nuxlli** [comment]
Hello, 

Answering your question: no, the project is NOT dead!

We do however understand how the seemingly lack of activities around it in recent months may have given that impression. 

We apologize for it. 

On the other hand, the truth is things couldn't be farther from that. 

In the last couple of months we've been very much involved in two types of less publicly visible activities: 

1 - creating revenue streams ($) for the project without messing with its open source qualities;

2 - new product version that will drastically expand its capabilities (including lowering the barrier for code contributions from the community).

The first step above was fundamental to assure the continuity of the project. It would have been impossible to move forward with the second step without it. 

We're still in the early phases of both to make any further public comments but rest assured that **azk** has a good future ahead. It will only take a little more patience (but just a little!).

As for the product support, we're resuming our regular activities more closely (chat support on gitter and github issues). 

Sorry again for making it look like as though azk was in a halt. We're back! ;) 

Thanks for caring to ask.

**Comment by chiaretto** [comment]
Hello,

Azk is a very useful tool, and very practical and would be very bad if it ended.

Congratulations to the whole team for the project.

I'm hoping that everything works out.

Thanks for the answer.

**Comment by elbouillon** [comment]
Hi, I'm asking another time, is azk still alive ?

**Comment by m9dfukc** [comment]
Hi, I'm also be very interested in an answer to this question! Huge part of our system is relying on azk right now. Hope there's a future for azk!

**Comment by FDiskas** [comment]
:swimmer:

**Comment by slobo** [comment]
Hi folks. First, thank you for azk and all the hard work you did on it. It is a central part of our development activities.
Could you shed some light on the future of azk? What are the chances of you continuing development? Or maybe you are interested in passing the project to be maintained by community somehow - maybe interested parties should create a fork?

Thanks

**Comment by spetrey** [comment]
This project seems dead... why is the DNS on azk.io non-resolvable? Has development been abandoned?

**Comment by slobo** [comment]
Looks like domain is expired. If there is no intention of renewing, perhaps azk team can give us epp transfer codes so azk.io doesn't go to spammers after the grace period?

**Comment by spetrey** [comment]
I agree. Mainly because of the documentation. Now that it has expired, the documentation is only visible via [wayback machine.](http://web.archive.org/web/20170606130142/http://docs.azk.io/) The docs need to live somewhere.

**Comment by slobo** [comment]
Also would be nice to get write access to this repo, we could also get github to host the docs using `gh-pages` branch...

**Comment by nuxlli** [comment]
Hello again,

First things first: the https://azk.io domain is out due to a small mistake in its renewal process. That’s all.

We expect to have it back again very soon. We apologize for the inconvenience.

Meanwhile, you can find our documentation online here: http://azk-docs.azukiapp.com/.

This had nothing to do with the project’s current status.

### Present and Future

It’s been about a year since I last posted on this thread. You can read it here: https://github.com/azukiapp/azk/issues/696#issuecomment-262964504.

Well, things took a little more time and effort than we had anticipated by then.

The good news is we got to a solution and we plan to announce it in about 30 days.

This time we got it.

When we first built `azk`, we also had to build several supporting tools around it ourselves, e.g. [**cli-router**](https://github.com/azukiapp/cli-router) and [**i18n-cli**](https://github.com/azukiapp/i18n-cli). 
 
Along with building `azk` itself, that was a bit hard on our fledgling team. 

Today we already have lots of tools that enable richer use cases such as [**Hyper_**](https://hypercontainer.io/) and [**Kubernetes**](https://kubernetes.io/) and that will save us a lot of time crafting some of the new features for `azk`. 

Besides we will implement a new OSS model that is less about a **“benevolent dictator”** and more **community driven** (e.g. using RFCs). 

We hope that will gradually make the project’s existence and evolution less and less dependent on just us.

For starters, we will be releasing a **new architecture** along with our announcement next month upon which we’ll start to encourage more and more code contributions.

Finally, when we started **Azuki** (our company behind `azk`), our motivation was empowering **individual programmers**.

Also, we wanted to make our tools free for them. Forever.

At that time, containers and tools like `azk` seemed to be the best way to achieve this goal. We have found since then that we can do more beyond `azk` itself.

With our new plans, we hope we’ll get closer to make programmers more independent and effective… this time with no interruptions along the way.

Thanks for all the interest in our little project.

Best.

**Comment by nuxlli** [comment]
http://azk.io is back online... sorry for the inconvenience ;)

**Comment by gerardo-junior** [comment]
Esse projeto realmente morreu?

**Comment by slobo** [comment]
Hey guys, `azk` has bit-rotted quite a bit. It's now impossible to even do a clean compile on MacOS 10.14.1 (on 10.14.0 compile worked). Is it safe to say that project is abandoned? I would appreciate the honesty, we invest quite a bit of time on regular basis working with the current bugs and quirks of `azk`, it would really help us decide how to move forward if we knew if there would be any further development.

If project is abandoned and/or seeking other maintainers, please put up a notice so that people can make informed decisions.

Thanks again for all the hard work you put into `azk`, and we really found it useful in the past, we still live in hope that things will turn back around.

Cheers

**Comment by gullitmiranda** [comment]
hi @slobo. we have not been able to invest time in the `azk`, even though we still believe that the space to which the `azk` proposed was not filled in. I personally can't transfer the maintenance of the project. But creating a fork is recommended and may expedite the addition of new maintainers.

tks for the appreciation
